package.json

```diff
@@ -140,7 +147,10 @@
     "survey-react": "^1.1.7",
     "svg-url-loader": "3.0.0",
     "time-input-polyfill": "^1.0.7",
+    "ts-loader": "^8.0.14",
     "twix": "1.3.0",
+    "typescript": "^4.1.3",
+    "typescript-eslint": "^0.0.1-alpha.0",
```

Comment:
This is a really old version of `typescript-eslint`. It seems that they are at `v4.14.0`.

Reply:
Yah I think this one was installed by mistake, this is https://www.npmjs.com/package/typescript-eslint but we want https://www.npmjs.com/package/@typescript-eslint/eslint-plugin.  I'll update